src/go/k8s/pkg/resources/certmanager/type_helpers.go

```diff
@@ -483,6 +587,28 @@ func (ac *apiCertificates) nodeCertificateName() *types.NamespacedName {
 	return nil
 }
 
+func (ac *apiCertificates) clientCACertificateName() *types.NamespacedName {
+	if ac.externalClientCACertificate == nil {
+		return nil
+	}
+	return &types.NamespacedName{
+		Name:      ac.externalClientCACertificate.Name,
+		Namespace: ac.externalClientCACertificate.Namespace,
+	}
+}
+
+// bundledClientCACertificateName derives the name for the bundled client CA certificate name
+// from the provided externalClientCACertificate.
+func (ac *apiCertificates) bundledClientCACertificateName() *types.NamespacedName {
+	if ac.externalClientCACertificate == nil {
```

Comment:
so what happens if external is not provided but we have only the non-generated. Is that still mounted properly? I bet yes, just asking out loud :D

Reply:
In that case, the "bundled" CA will only be one CA cert. I'll make sure we cover that scenario in the kuttl tests.